### ISSUE TITLE
Remove app billing and donation related code

### ISSUE THREAD

**Issue by syt0r** [SYtor/WifiPass#7]
I received almost no donations and  I can't publish new version on google play because of outdated billing library, so it would be nice if someone removed everything billing related from application and increased version of the app

**Comment by SynapseScribe** [comment]
now that app is working for newer android versions, maybe it will get more traction. However, i agree removing google related code to keep the app clean. Maybe you can switch to other donation strategies such as on github, patreon, crypto etc. Also the app should be advertised on android platforms such as xda forums to gain visibility, if you'd wish to monetize it.

**Comment by syt0r** [comment]
I'm not very enthusiastic about further support of the app since password sharing functionality is available in newer android versions out of the box even without root access but I welcome contributions if someone is interested in it, maybe for coding practice purposes

**Comment by SynapseScribe** [comment]
Could you at least push v3 to playstore? Thanks!

**Comment by syt0r** [comment]
No, google doesn't let you update an app with a deprecated billing library unfortunately

**Comment by SynapseScribe** [comment]
Then maybe better alternative is fdroid
